### ISSUE TITLE
Meilenstein 1: erste spielbare Version

### ISSUE THREAD

**Issue by SprachRolf** [PaulStahr/SimpleBoardGameSimulator#13]
Was brauchen wir denn noch, bis SimpleBoardGame in Alpha-Version spielbar ist?
- Darstellung der Ha:nde der Gegenspieler
- 2 Wu:rfel
- Ein Knopf fu:r Karten mischen und austeilen
- noch etwas?

**Comment by PaulStahr** [comment]
Ich würde 2 Würfel zu Würfel ändern, da die Anzahl nur von der Spieldatei abhängt und nichts mit der Implementierung zu tun hat

**Comment by fseiffarth** [comment]
- Ich würde vorschlagen, dass wir die erste Version ohne Hände der Gegenspieler machen. Die Karten der Gegenspieler sieht man schon. Ich hatte darüber nachgedacht die Hände der Gegenspieler kleiner am oberen Rand optional einzublenden.

- Würfel habe ich hinzugefügt. 

- Karten kann man mit S mischen. Ich weiß nicht genau wie wir einen Knopf zum austeilen machen können. Bisher funktioniert das so, dass man Karten zu einem Stapel zusammenfügen kann und über das Mausrad dann n-Karten von diesem Stapel ziehen kann.

**Comment by SprachRolf** [comment]
Ja, du hast Recht, es reicht für die erste spielbare Version, wenn man die eigene Hand sieht :-)
Karten per Mausrad ziehen ist auch schon spielbar.
